apps/prairielearn/src/components/EditQuestionPointsScore.tsx

```diff
@@ -136,3 +144,297 @@ function EditQuestionPointsScoreForm({
     </form>
   `;
 }
+
+// Preact version with mutation support
+interface EditQuestionPointsMutationParams {
+  instance_question_id: string;
+  modified_at: string;
+  field: 'points' | 'auto_points' | 'manual_points' | 'score_perc';
+  value: number | null;
+}
+
+type EditQuestionPointsMutationResponse = {
+  conflict_grading_job_id: string;
+  conflict_details_url: string;
+} | null;
+
+export function useEditQuestionPointsMutation({ csrfToken }: { csrfToken: string }) {
```

Comment:
This is exported but I don't see it being used anywhere.

Reply:
Someday `knip` can solve this for us :)

Reply:
Addressed in be92cde5ad156bcbabaec1ae7e00a43059e5e6cd or [1cfcaa3](https://github.com/PrairieLearn/PrairieLearn/pull/13117/commits/1cfcaa3e8c1c075098474646f901e105dd58a963).